diagrams/duplex-messaging/duplex-creating.mmd

```diff
@@ -16,38 +16,37 @@ sequenceDiagram
   AS ->> AA: IDS: SMP queue IDs
   note over AA: status: NEW/NONE
 
-  A ->> AA: INVITE: invite<br>another party
-  AA ->> A: JOIN: invitation<br>to connect
+  AA ->> A: INV: invitation<br>to connect
   note over AA: status: PENDING/NONE
 
   note over A, B: 3. out-of-band invitation
   A ->> B: OOB: invitation to connect
-  B ->> BA: CONNECT:<br>via invitation info
+
+  note over BA, B: 4. accept connection
+  B ->> BA: JOIN:<br>via invitation info
   note over BA: status: NONE/NEW
 
-  note over BA, AA: 4. establish Alice's SMP queue
+  note over BA, AA: 5. establish Alice's SMP queue
   BA ->> AS: SEND: KEY: sender's key and Bob's profile
   note over BA: status: NONE/CONFIRMED
```

Comment:
since there's mention of Bob's and Alice's profiles being sent in steps 5 and 7, should there be steps where agents send profiles to Alice and Bob?

Reply:
we don't send profiles now...